### PR TITLE
adding option to add extra RPMs during install

### PR DESCRIPTION
**Pull request by oblandford-novetta**

### CODE REVIEW

File: OSpackages.sh [hunk 1]
```diff
@@ -318,8 +331,8 @@ function FetchCustomRepos {
 ## Main program-flow
 ######################
 OPTIONBUFR=$( getopt \
-   -o a:Fg:hm:r:M: \
-   --long help,mountpoint:,repo-activation:,repo-rpms:,rpm-group: \
+   -o a:eg:hm:r:M: \
```
Comment (lorengordon) [inline OSpackages.sh:334]: Huh. I wonder what `F` was supposed to be? @ferricoxide?

File: OSpackages.sh [hunk 2]
```diff
@@ -99,7 +99,8 @@ function UsageMsg {
       printf '\t%-4s%s\n' '-a' 'List of repository-names to activate'
       printf '\t%-6s%s' '' 'Default activation: '
       GetDefaultRepos
-      printf '\t%-4s%s\n' '-g' 'RPM-group to intall (default: "core")'
+      printf '\t%-4s%s\n' '-e' 'Extra RPMs to install '
```
Comment (lorengordon) [inline OSpackages.sh:102]: Let's add the long option below also, `--extras`

File: OSpackages.sh (hunk 1 above)
Comment (ferricoxide) [reply OSpackages.sh:334]: I'm assuming copy-paystah error:

~~~
$ grep -- -F *
DiskSetup.sh:                  MKFSFORCEOPT="-F"
PostBuild.sh:      printf '\t%-4s%s\n' '-F' 'Disable FIPS support (NOT IMPLEMENTED)'
PostBuild.sh:      printf '\t%-20s%s\n' '--no-fips' 'See "-F" short-option'
PostBuild.sh:      -F|--no-fips)
~~~

**Comment by oblandford-novetta** [comment]
Closing as I have more troubleshooting to do.

**Pull request closed without merging**